opencog/combo/converter/combo_atomese.cc

```diff
@@ -125,6 +130,11 @@ std::pair<Type, Handle> vertex_2_atom::operator()(const builtin &b) const
 	return std::make_pair(type, Handle());
 }
 
+std::pair<Type, Handle> vertex_2_atom::operator()(const contin_t& c) const
+{
+	return std::make_pair(-1, Handle(createNumberNode(c)));
+}
+
 std::pair<Type, Handle> vertex_2_atom::operator()(const enum_t &e) const
 {
 	return pair<Type, Handle>();
```

Comment:
That would be a case where `{}` comes handy, less typing
```suggestion
	return {};
```

Reply:
should I change it to this format?